src/wirecloud/defaulttheme/templates/registration/login.html

```diff
@@ -28,7 +28,7 @@
 
         <div id="unsupported-browser-msg" class="alert alert-error" style="display: none; text-align: left;">
             <h4>{% trans "Your browser seems to lack some required features" %}</h4>
-            <p>{% blocktrans %}We recommend you to upgrade your browser to the newest version of either <a href="https://www.mozilla.org/firefox">Firefox</a> or <a href="www.google.com/chrome">Google Chrome</a> as these are the browsers currently supported by WireCloud.{% endblocktrans %}</p>
```

Comment:
Found by chance that the link to Chrome was not working because of the missing protocol (tried to open in Internet Explorer)